lib/macho/headers.rb

```diff
@@ -61,17 +69,69 @@ module MachO
   # @see http://llvm.org/docs/doxygen/html/Support_2MachO_8h_source.html
   CPU_SUBTYPE_LIB64 = 0x80000000
 
-  # all x86-type CPUs
+  # all CPU sub-types for CPU type `CPU_TYPE_I386`
   CPU_SUBTYPE_X86_ALL = 3
 
-  # all x86-type CPUs (what makes this different from CPU_SUBTYPE_X86_ALL?)
-  CPU_SUBTYPE_X86_ARCH1 = 4
+  # all CPU sub-types for CPU type `CPU_TYPE_X86_64`
+  CPU_SUBTYPE_X86_64_ALL = CPU_SUBTYPE_X86_ALL
+
+  # all CPU sub-types for CPU type `CPU_TYPE_X86_64` (Haskell tree)
+  CPU_SUBTYPE_X86_64_H = 8
```

Comment:
I had them separate because YARD only documents them correctly when separated. That was a personal preference, so I'm happy to change it if you prefer.